### PR TITLE
Adds "Rewrite Loadout" button to the loadout vendor.

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Ports https://github.com/Cosmic-Overlord/TerraGov-Marine-Corps/pull/35
image
image
image
## Why It's Good For The Game
It's just better loadout management.
## Changelog
:cl:
add: "Overwrite loadout" button in Loadout Vendor
/:cl:

### CODE REVIEW

**Comment by maikilangiolo** [comment]
No fucking way. We have the technology

File: code/modules/tgui_input/alert.dm [hunk 1]
```diff
@@ -59,12 +59,14 @@
 	var/autofocus
 	/// Boolean field describing if the tgui_alert was closed by the user.
 	var/closed
+	var/src_object
 
-/datum/tgui_alert/New(mob/user, message, title, list/buttons, timeout, autofocus)
+/datum/tgui_alert/New(mob/user, src_object, message, title, list/buttons, timeout, autofocus)
```
Comment (TiviPlus) [inline code/modules/tgui_input/alert.dm:64]: ?? what is this

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/tgui_input/alert.dm (hunk 1 above)
Comment (Xander3359) [reply code/modules/tgui_input/alert.dm:64]: idk I've just ported it from RU

**Comment by zzurge** [comment]
I kneel

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**